.pre-commit-config.yaml

```diff
@@ -45,7 +45,7 @@ repos:
         # This doesn't work well with `mypy --follow-imports error`
         # See: https://github.com/pre-commit/mirrors-mypy/issues/34#issuecomment-1062160321
         #
-        # To workaround this we run `mypy` only in manual mode
+        # To work around this we run `mypy` only in manual mode
```

Comment:
Ah, while `workaround` is a noun, it isn't a verb, so this makes sense.

Reply:
Yes, this is exactly what PyCharm was telling 👍